tasmota/tasmota_xdrv_driver/xdrv_79_esp32_ble.ino

```diff
@@ -2479,7 +2479,7 @@ int extQueueOperation(BLE_ESP32::generic_sensor_t** op){
   }
 
   if (!BLEMasterEnable){
-    AddLog(LOG_LEVEL_ERROR,PSTR("BLE: extQueueOperation: BLE is deiabled"));
+    AddLog(LOG_LEVEL_ERROR, PSTR("BLE: extQueueOperation: BLE is disabled"));
```

Comment:
This change does not belong in the MR.

Reply:
Found while testing, so fixed it directly.